### PR TITLE
LOC creation by requester and LO region

### PR DESCRIPTION
**Pull request by benoitdevos**
### LOC creation by requester
For LOC whose requester has a Polkadot address, flow of Acceptance/Creation is adapted
* LLO Owner just need to accept - no fees associated.
* The requester self-opens the LOC and thus is charged for Inclusion fees (and Legal Fees too).
* A new tab is present for both LLO and requester: "Accepted".
* An Accepted request can be cancelled (deleted), reworked or opened.
* Use new status values: `REVIEW_PENDING`, `REVIEW_REJECTED`,  `REVIEW_ACCEPTED`.

### LO region
Region is displayed in settings and view
A select control is used in settings but it will more than probably be disabled all the time as LOs are assigned a region as soon as they join and cannot change it afterwards.
___
Includes the following PR's:
https://github.com/logion-network/logion-wallet/pull/479
https://github.com/logion-network/logion-wallet/pull/477
https://github.com/logion-network/logion-wallet/pull/478
___
logion-network/logion-internal#911

### CODE REVIEW

File: src/common/LocStatusCell.test.tsx [hunk 1]
```diff
@@ -3,14 +3,14 @@ import LocStatusCell from './LocStatusCell';
 
 it("renders REQUESTED", () => {
     const tree = shallowRender(<LocStatusCell
-        status="REQUESTED"
+        status="REVIEW_PENDING"
     />);
     expect(tree).toMatchSnapshot();
 });
 
 it("renders REJECTED", () => {
```
Comment (gdethier) [inline src/common/LocStatusCell.test.tsx:11]:
```suggestion
it("renders REVIEW_REJECTED", () => {
```

File: src/legal-officer/transaction-protection/LocRequestAcceptance.tsx [hunk 2]
```diff
@@ -118,7 +118,7 @@ export default function LocRequestAcceptance(props: Props) {
                             });
                         } else {
                             const apiLimits = await toApiLimits(client.logionApi, limits);
-                            return current.legalOfficer.acceptCollection({
+                            return current.legalOfficer.accept({
```
Comment (gdethier) [inline src/legal-officer/transaction-protection/LocRequestAcceptance.tsx:121]: Handled at SDK side?

File: src/wallet-user/transaction-protection/__snapshots__/DraftLocs.test.tsx.snap [hunk 3]
```diff
@@ -253,33 +253,7 @@ exports[`DraftLocs renders pending requests 1`] = `
...
-                  "color": "#ff9b3f",
-                  "textTransform": undefined,
-                }
-              }
-            >
-              Pending
-            </span>
-          </span>
-        </div>
+        />
```
Comment (gdethier) [inline src/wallet-user/transaction-protection/__snapshots__/DraftLocs.test.tsx.snap:256]: Not sure this is expected.

File: src/wallet-user/transaction-protection/__snapshots__/RequestedLocs.test.tsx.snap [hunk 4]
```diff
@@ -231,33 +231,7 @@ exports[`RequestedLocs Renders pending requests 1`] = `
...
-                  "color": "#ff9b3f",
-                  "textTransform": undefined,
-                }
-              }
-            >
-              Pending
-            </span>
-          </span>
-        </div>
+        />
```
Comment (gdethier) [inline src/wallet-user/transaction-protection/__snapshots__/RequestedLocs.test.tsx.snap:234]: Expected?

**Review by gdethier — APPROVED** [review]
Good job so far. I have a couple of questions/remarks.

File: src/legal-officer/transaction-protection/LocRequestAcceptance.tsx (hunk 2 above)
Comment (benoitdevos) [reply src/legal-officer/transaction-protection/LocRequestAcceptance.tsx:121]: Yes.

**Pull request merged**